docs/content/en/docs/tasks/cluster/cluster-upgrades/baremetal-upgrades.md

```diff
@@ -16,12 +16,24 @@ The upgrade command also upgrades core components of EKS Anywhere and lets the u
 >**_NOTE:_** Currently only Minor Version Upgrades are support for Bare Metal clusters. No other aspects of the cluster upgrades are currently supported.
 >
 
+### Modular Cluster Upgrades
```

Comment:
I don't think we need a separate section for this. Instead I would like to call out in a "Note" that mentions previous versions of EKS-A (v0.13 and below?) had a bug that could cause workload clusters to upgrade when a management cluster EKS-A version upgrade was triggered. Now that no longer happens.